packages/tools/pie-icons-vue/icons/IconCheckCircleLarge.js

```diff
@@ -3,13 +3,13 @@ import _mergeJSXProps from '@vue/babel-helper-vue-jsx-merge-props';
 import { iconSize, updateContextData } from './configs-vue';
 
 export default {
-    name: 'IconCheckCirclLarge',
+    name: 'IconCheckCircleLarge',
```

Comment:
Should this be `checkedCircle` to make it more explicitly say what the icon represents?

Reply:
Interesting question. I would say no because it is a circle with a check in it. "Checked" would imply the existence of an unchecked variant, to me at least.